std_msgs/Header header
geometry_msgs/Twist velocity
float64 battery_level
float64 distance_traveled
geometry_msgs/Pose coordinates
builtin_interfaces/Time travel_time

